### DISCUSSION TITLE
Mirth Connect 4.4.2

### DISCUSSION THREAD

**Discussion by pladesma** [nextgenhealthcare/connect#5977] [General]
Mirth Connect 4.4.2 is a patch release that includes bug fixes and an improvement to exporting messages.

[See What's New](https://github.com/nextgenhealthcare/connect/wiki/4.4.2---What's-New) | [Upgrade Guide](https://github.com/nextgenhealthcare/connect/wiki/Upgrading) | [Release Notes](https://github.com/nextgenhealthcare/connect/milestone/82?closed=1)

# Downloads

### Connect

OS | Arch | Download Link | SHA256 | MD5
-- | ---- | ------------- | ------ | ---
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-windows-x32.exe) | `e01732bab9766e985156d05e38290dd3a6e3f5293e366d5878362c7e9ce2b5b0` | `fd52572cd88e778d2f508b72ac8a3147`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-windows-x32.zip) | `eea17f7e6d7a79b069ef6c4e883d465aa24283a4af54b7929833eba4bd7c79d3` | `f0ce63a629d365e6359b8fd344f2e926`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-windows-x64.exe) | `415fc52948df4d64296bf3b89b47379ee4766fabc0fc2db357c9b1698a45a83d` | `c8b488ff69dab6f9cee4e85bfc2e8bce`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-windows-x64.zip) | `ad4ebacacea757e36021fd0a5078b7695cee3edd56c71ac11e8d252dcfd3041e` | `682e8175ad4cf0cde89fe0fbefbac286`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [RPM](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-linux.rpm) | `55f8bdd2b0974676c095e0ab19fe26b2e7b67c9caf9abc9819d8d1b0a832b2b7` | `b269f592eb529bb28e18e4db7413f589`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-unix.tar.gz) | `303bab13f6defaa0f3af16a474fb91b3c178c10b11a614aff5e78bc739032067` | `ebfbd7a1f2806e270ae2defcbf6d2351`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-unix.sh) | `34065cc44e19cc694c911396c7b26dfdd1b0a1334bb8755bc4d1c89837766229` | `3e1439181a6f556545114747fef2c95c`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-macos.dmg) | `881aa1f92b777060c2e0bcc09c173a5b6158815325868758213c42a1837060a7` | `b52515f523a52d450b44bd323b306af2`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnect-4.4.2.b326-unix.tar.gz) | `303bab13f6defaa0f3af16a474fb91b3c178c10b11a614aff5e78bc739032067` | `ebfbd7a1f2806e270ae2defcbf6d2351`

### Command Line Interface

OS | Arch | Download Link | SHA256 | MD5
-- | ---- | ------------- | ------ | ---
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnectcli-4.4.2.b326-windows-x32.zip) | `d637beec6ea0689c02178aa8c63a0aba61183767ef2e685da659462f031278ea` | `5084f0710b9ea105a43249a672804ebd`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnectcli-4.4.2.b326-windows-x64.zip) | `9c7c1b7fea37042b8251d2efa87493fe8462143c849cd99eb460fdb141998971` | `bb9317453982698ceb43b08bfb473c1c`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnectcli-4.4.2.b326-unix.tar.gz) | `29eb3a712641663c91877d97b465f7432ded76adff0bb307395070de67afd035` | `759a402ae8c3e7406c9fb7965060512c`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.2.b326/mirthconnectcli-4.4.2.b326-unix.tar.gz) | `29eb3a712641663c91877d97b465f7432ded76adff0bb307395070de67afd035` | `759a402ae8c3e7406c9fb7965060512c`

<hr /><em>This discussion was created from the release <a href='https://github.com/nextgenhealthcare/connect/releases/tag/4.4.2'>Mirth Connect 4.4.2</a>.</em>

**Comment by nosbodyevad** [comment]
Is 4.4.2 CURES certified? On the last call it was stated that it would be, but wanted to verify. Thanks!
